src/Kygekraqmak/KygekPingTPS/Main.php

```diff
@@ -13,55 +13,56 @@
  *          |____/ |____/                           |_|
  *
  * A PocketMine-MP plugin to see the server TPS and a player's ping
- * Copyright (C) 2020-2021 Kygekraqmak, KygekTeam
+ * Copyright (C) 2020-2021 Kygekraqmak
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
  *
-*/
+ */
 
 declare(strict_types=1);
 
 namespace Kygekraqmak\KygekPingTPS;
 
```

Comment:
i alway put the dependents to the first lol

Reply:
> i alway put the dependents to the first lol

What do you mean? Please explain.